src/js/helpers/cleanData.js

```diff
@@ -2,13 +2,15 @@ import { findKey } from 'lodash';
 import filesize from 'filesize';
 import axios from 'axios';
 import moment from 'moment-timezone';
+import Cookies from 'js-cookie';
 import { parsePhoneNumberFromString } from 'libphonenumber-js';
 
 import { WEEKDAY_MAP } from 'js/helpers/constants';
 
 export const formatTime = time => {
+  let lang = Cookies.get('lang')
   if (time === '12:00:00') {
-    return 'Noon';
+    return lang === 'es' ? 'Mediodía' : 'Noon';
   }
```

Comment:
is there a better way to handle the localization outside of React components? This obviously doesn't handle our other languages, but I figured for MVP....

Reply:
You can use the `.json`s we have for translations in `https://github.com/cityofaustin/janis/blob/master/src/js/i18n/locales`

Reply:
I'd still be hardcoding it though, right? `lang === 'es' ? 'date.Noon' : 'Noon'`

Reply:
@chiaberry I don't think so, it'd be more like
```
if (time === '12:00:00') {
  return intl.formatMessage(noonKey)
}
```